packages/ui-extensions/src/surfaces/point-of-sale/api/postpurchase-api/postpurchase-api.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Interface to access the order after checkout
+ */
+export interface PostPurchaseAPI {
```

Comment:
I think we should name this `OrderAPIContent`, which will be generic enough to be used in OrderDetails down the road. What do you think? The Content part is strictly for the interface naming as this is how we have all our other interfaces defined (you can look up CartAPIContent for example).